spec/models/lettings_log_spec.rb

```diff
@@ -2797,6 +2797,130 @@ def previous_postcode_fields
         expect(described_class.filter_by_user(%w[all yours], created_by_user).count).to eq(3)
       end
     end
+
```

Comment:
can you add some tests for `duplicate_logs_for_organisation` too? same in `sales_log_spec` :)